### ISSUE TITLE
UIIconButton symbol not centered horizontally in Safari on MacOSX

### ISSUE THREAD

**Issue by gdepourtales** [JosephusPaye/Keen-UI#48]
Hi,

First of all, your package is awesome. Many thanks for sharing it with the community. 

I have a spotted a small issue with UIIconButton which does not center horizontally the icon in the button on Safari MacOSX. See screenshot from the documentation.
<img width="335" alt="screen shot 2016-06-01 at 22 15 33" src="https://cloud.githubusercontent.com/assets/810628/15724298/bb1f9516-2846-11e6-905e-92d2ebf61d46.png">

Actually, centering the icon can be achieved by changing the width in the `ui-icon-button-icon` class to 100% which forces the horizontal centering of the icon:

```css
.ui-icon-button-icon {
    width: 100%;
    height: initial;
}
```
I have successfully tested it on latest versions of Safari, Chrome and Firefox on MacOSX.

**Comment by JosephusPaye** [comment]
Hey @gdepourtales, 

Thanks for the fix, it will be added in v0.8.9.

**Comment by gdepourtales** [comment]
Many thanks @JosephusPaye

**Comment by gdepourtales** [comment]
BTW, I experienced the same issue on UIFab and fixed like this

```css
.ui-fab-icon {
    margin: 0;
    width: 100%;
    height: initial;
}
```

**Comment by JosephusPaye** [comment]
The fix is now available in [v0.8.9](https://github.com/JosephusPaye/Keen-UI/releases/tag/v0.8.9).

**Comment by vadimsg** [comment]
I have v0.8.9 and I have some problems with icons on ipad and iphone safari. The icons are not centered. I fixed it with this simple rule:

.ui-icon-button {
  padding: 0;
}
